tests/TestCase.php

```diff
@@ -1,21 +1,13 @@
-<?php
+<?php declare(strict_types=1);
 
 namespace Stylers\TaskManager\Tests;
 
 use Illuminate\Foundation\Application;
 use Orchestra\Testbench\TestCase as BaseTestCase;
 use Stylers\TaskManager\Providers\TaskManagerServiceProvider;
 
-/**
- * Class TestCase
- * @package Stylers\TaskManager\Tests
- */
 abstract class TestCase extends BaseTestCase
 {
-    /**
-     * @param Application $app
-     * @return array
-     */
     protected function getPackageProviders($app): array
```

Comment:
`protected function getPackageProviders(Application  $app): array`?

Reply:
Not compatible with parent class method